resources/views/contact/contact_form.blade.php

```diff
@@ -11,10 +11,14 @@
                             <form action="/contact" method="POST">
                                 @csrf
                                 <h1 class="mb-4">Contact Us</h1>
-                                @if (session()->get('success'))
+                                @if (session()->get('success') === true)
                                     <div class="p-4 mb-4 alert-success">
                                         {{'Email sent successfully. We\'ll get back to you as soon as possible.'}}
                                     </div>
+                                @elseif (session()->get('success') === false)
+                                    <div class="p-4 mb-4 alert-danger">
+                                        {{'Something went wrong. Please try again.'}}
```

Comment:
You can stop relying on success here and check for the error itself once the changes to the controller above are made.
```suggestion
                                @elseif ($errors->has('recaptcha'))
                                    <div class="p-4 mb-4 alert-danger">
                                        {{ $errors->first('recaptcha') }}
```